src/app/collection/pages/collection-ncyte/feature/collection-learning-object-card/collection-learning-object-card.component.html

```diff
@@ -0,0 +1,33 @@
+<div class = "container">
+
+    <div *ngIf = "learnObj.length !== 'micromodule'">
+        <div class="module-background">
+            <img class ="background"src="../../../../../../../assets/images/michael-dziedzic-nLFqr9Mr9H8-unsplash 1.png" alt="image not shown">
+            <div class = "module">MODULE</div>
```

Comment:
This should come directly from the learningObject itself. learningObject.length is the property you'll need to access.

Reply:
this was from the old commit too, the latest commit doesnt have this anymore

Reply:
I'm still seeing this. Could you push again?